File: ELIMINACION INICIO,ESP,FINAL/eliminacion.js
const originalarr=[11,21,31,41,51,61];

console.log("eliminacion al inicio");
let inputarr=[...originalarr];
let position=0;

console.log("antes: ",inputarr);
for (let n=position;n<inputarr.length-1;n++){
    inputarr[n]=inputarr[n+1];
}
inputarr.pop();
console.log("despues: ",inputarr);
console.log("");

//eliminacion especifico
console.log("eliminacion especifico");
inputarr=[...originalarr];
position=3;
console.log("antes: ",inputarr);
for (let n=position;n<inputarr.length-1;n++){
    inputarr[n]=inputarr[n+1];
}   
inputarr.pop();
console.log("despues: ",inputarr);
console.log("");

//eliminacion al final
console.log("eliminacion al final");
inputarr=[...originalarr];
console.log("antes: ",inputarr);
inputarr.pop();
console.log("despues: ",inputarr);

File: ELIMINACION INICIO,ESP,FINAL/eliminacion.py
original_arr=[11,21,31,41,51,61]

#liminacion al principio
print("eliminacion al inicio: ")
inputarr=list(original_arr)
position=0

print("antes: ",inputarr)
for f in range(position,len(inputarr)-1):
    inputarr[f]=inputarr[f+1]
inputarr.pop()
print("despues: ",inputarr)
print()
#eliminacion especifico
print("eliminacion especifico: ")
inputarr=list(original_arr)
position=3

print("antes: ",inputarr)
for f in range(position,len(inputarr)-1):
    inputarr[f]=inputarr[f+1]
inputarr.pop()
print("despues: ",inputarr)
print()


#eliminacion al final
print("eliminacion al final: ")
inputarr=list(original_arr)
print("antes: ",inputarr)
inputarr.pop()
print("despues: ",inputarr)
